src/overrides/hooks/lists.py

```diff
@@ -1,256 +1,204 @@
+            lambda match: replace_content(match, "instrumenten"),
+            file.page.content,
+            flags=re.I | re.M,
         )
 
-        # Find and replace all strings in current page to list of vereisten on maatregelen page
-        file.page.content = re.sub(
-            r"<!-- list_vereisten_on_maatregelen_page -->", replace_vereisten_on_maatregelen_page, file.page.content, flags=re.I | re.M
-        )
-
-def _create_table_row(file: File):
+def _create_chip(item: str, link: str, chip_type: str) -> str:
+    if not item:
+        return ""  # Return an empty string if the item is None or empty
+
+    if chip_type == 'rol':
+        color_class = 'deep-orange'
+        icon_svg = '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 4a4 4 0 0 1 4 4 4 4 0 0 1-4 4 4 4 0 0 1-4-4 4 4 0 0 1 4-4m0 10c4.42 0 8 1.79 8 4v2H4v-2c0-2.21 3.58-4 8-4Z"></path></svg>'
+    elif chip_type == 'levenscyclus':
+        color_class = 'indigo'
+        icon_svg = '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M2 12a9 9 0 0 0 9 9c2.39 0 4.68-.94 6.4-2.6l-1.5-1.5A6.706 6.706 0 0 1 11 19c-6.24 0-9.36-7.54-4.95-11.95C10.46 2.64 18 5.77 18 12h-3l4 4h.1l3.9-4h-3a9 9 0 0 0-18 0Z"></path></svg>'
+    elif chip_type == 'onderwerp':
+        color_class = 'teal'
+        icon_svg = '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 2a7 7 0 0 0-7 7c0 2.38 1.19 4.47 3 5.74V17a1 1 0 0 0 1 1h6a1 1 0 0 0 1-1v-2.26c1.81-1.27 3-3.36 3-5.74a7 7 0 0 0-7-7M9 21a1 1 0 0 0 1 1h4a1 1 0 0 0 1-1v-1H9v1Z"></path></svg>'
+    else:
+        color_class = 'blue'
+        icon_svg = '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M2 12a9 9 0 0 0 9 9c2.39 0 4.68-.94 6.4-2.6l-1.5-1.5A6.706 6.706 0 0 1 11 19c-6.24 0-9.36-7.54-4.95-11.95C10.46 2.64 18 5.77 18 12h-3l4 4h.1l3.9-4h-3a9 9 0 0 0-18 0Z"></path></svg>'
+    
+    return f'''
+        <span class="mdx-badge" data-md-color-accent="{color_class}">
+            <span class="mdx-badge__icon">
+                <a href="{link}" title="{item}">
+                    <span class="twemoji">{icon_svg}</span>
+                </a>
+            </span>
+            <span class="mdx-badge__text">
+                <a href="{link}">{item}</a>
+            </span>
+        </span>
+    '''
+
+def _create_table_row_2(file: File, filter_options: Dict[str, bool]) -> str:
+    # Ensure metadata exists or is defaulted to empty list
+    rollen = file.page.meta.get('rollen', [])
+    levenscyclus = file.page.meta.get('levenscyclus', [])
+    onderwerpen = file.page.meta.get('onderwerp', [])
+
+    # Create chips conditionally based on filter options
+    rollen_chips = ''.join(
+        _create_chip(rol, f"../../rollen/{rol}.md", 'rol') for rol in rollen
+    ) if filter_options.get("rol", True) else ""
+
+    levenscyclus_chips = ''.join(
+        _create_chip(lc, f"../../levenscyclus/{lc}.md", 'levenscyclus') for lc in levenscyclus
```

Comment:
het probleem met de kapotte links zit hier. We gaan twee mappen terug, en dat klopt wanneer je vereisten wilt tonen op bijvoorbeeld de [pagina van een onderwerp](https://minbzk.github.io/Algoritmekader/pr-preview/pr-259/onderwerpen/bias-en-non-discriminatie/#vereisten).

Op de pagina van [vereisten](https://minbzk.github.io/Algoritmekader/pr-preview/pr-259/vereisten/) wil je maar 1 map omhoog. Als ik daar op een vereiste klik zie je ook dat er een stuk van de URL ontbreekt, zie bijv. https://minbzk.github.io/Algoritmekader/pr-preview/vereisten/CE_markering_hoog_risico/index.html

Lokaal gaat dit op de een of andere manier wel goed, maar in de preview (en ook na een merge heb ik uit ervaring geleerd) helaas niet. Moeten we dus toch nog een oplossing voor verzinnen.

Die code was heel lelijk geworden, doordat ik dit de laatste keer heel snel gefixt had na een release